WORKSPACE

```diff
@@ -34,10 +34,11 @@ pip_import(
 load("@grpc_python_dependencies//:requirements.bzl", "pip_install")
 pip_install()
 
+# TODO(ghostwriternr): Switch to upstream repo
```

Comment:
Put a link to the issue you've filed against the upstream repository in place of your username - [we don't allow usernames in TODOs](https://github.com/grpc/grpc/blob/912b8ab4d49cd6cde76675d37c896e5edcf67487/.pylintrc#L16-L19) (though I'm sure you'll find lots of disallowed ones if you go looking for them).

Reply:
Done!